### PR TITLE
chore: had good description until linter didn't like it

### PR DESCRIPTION
**Pull request by bpedersen**
**Type:**

The following has been addressed in the PR:

*  There is a related issue?
*  Unit or Functional tests are included in the PR

**Description:**

### CODE REVIEW

File: src/pages/Version/Version.test.tsx [hunk 1]
```diff
@@ -18,8 +18,7 @@ describe('test Version page', () => {
   jest.setTimeout(40000000);
   beforeAll(() => {
     // FIXME: a better way to mock this
```
Comment (bpedersen) [inline src/pages/Version/Version.test.tsx]: We can declare this VERDACCIO_API_URL as a global variable in the jest config if we want this to always be 'http://test'

**Comment by priscilawebdev** [comment]
@bpedersen you have conflicts 😉

File: src/pages/Version/Version.test.tsx [hunk 2]
```diff
@@ -23,6 +23,22 @@ const detailContextValue = {
 };
 
 describe('test Version page', () => {
+  jest.setTimeout(40000000);
```
Comment (juanpicado) [inline src/pages/Version/Version.test.tsx]: Please remove this :-)

File: src/pages/Version/Version.test.tsx (hunk 1 above)
Comment (juanpicado) [reply src/pages/Version/Version.test.tsx]: Agree 👍

**Comment by DanielRuf** [comment]
Hey @bpedersen,

when do you have some time to take another look at this?
Also can we reword the first commit and PR? What exactly did the linter not like?

**Comment by priscilawebdev** [comment]
as this PR is inactive for a long period of time, I'm closing it 😉

**Pull request closed without merging**